less/mcq.less

```diff
@@ -28,6 +28,21 @@
   }
 }
 
+.mcq.can-show-correctness {
```

Comment:
For consistency with `_canShowModelAnswer`, instead of nesting styles within `.mcq.can-show-correctness`, should lines 34, 40 and 41 be prefixed `&__widget.can-show-correctness`.

For example:
`&__widget.can-show-correctness .mcq-item__answer-icon {`

Reply:
`can-show-correctness` is not a class on the widget, it's a class on the component container, it would be:
```css
.can-show-correctness &__widget .mcq-item__answer-icon {
```
Does that work?

Reply:
ah, I missed this. Maybe having the `.can-show-correctness` class on the widget instead would be more consistent. Unless there's good reason for this to be on the component container instead?

Reply:
Surprisingly, it's more complicated than it looks.

When `_canShowModelAnswer: true` the widget switches between `show-user-answer` and `show-correct-answer` when the button is toggled.
When `_canShowModelAnswer: false` the widget still needs the class `show-user-answer` as the less relies on that class for styling. 
I've used `show-correctness` for the new styles when `_canShowCorrectness: true`.